EIPS/eip-1023.md

```diff
@@ -0,0 +1,65 @@
+
+
+To preserve compatibility and expectations of `blockhash` from PoW to PoS, this EIP advises replacing PoS's `blockhash` with a similarly economically staked value; or, at the very least, exposing a new value named `block.entropyhash`.
+
+## Abstract
+In PoW, the `blockhash` is not knowable by a miner before first successfully hashing the block, which carries an economic cost. In this sense `blockhash` is used by contracts as a randomly distributed number under the assumption there must be some **_minimum_ economic incentive** required to exploit it.
+
+In PoS, the `blockhash` is not a source of entropy at all; a validator can submit a block whose `blockhash` results in any desired outcomes, with _zero_ cost. In this sense, `blockhash` can now be exploited by validators provided there is **_any_ economic incentive.**
+
+In PoW, the economic incentive to coerce `blockhash` had to be about `block reward - uncle reward`, in PoS, it merely needs to be `>0`. 
+
+Some example cases in PoW that are now vulnerable in PoS:
+
+ - Using `blockhash`, or any derivation of it, as a randomly distributed number for benign purposes; eg, distributing items into buckets.
+ - Using `blockhash`, or any derivation of it, to determine outcomes that had a very small amount of Ether associated with it.
+
+## Motivation
+There are three main motivations for providing an economically staked randomly distributed number:
+
+ - Preserve backward compatibility with PoW, which does provide such a number in the form of `blockhash`
+ - Prevent validators from wreaking havoc on anything that currently uses `blockhash` in any form.
+ - Provide a sanctioned value with clear specifications about when and how it can be used.
+
+A fourth reason is that providing a built-in source of randomness is a _very_ nice feature for contract developers. As long as the limitations of the use of this random number are made clear, it's a massive savings in effort as compared to using other random number systems (oracles, multiple signatures, etc). Furthermore, it provides a easier route to autonomy for contracts that _do_ require randomness -- all other random number schemes require an additional transaction to be executed, and depend on complex behavior. It would be very nice to have a random number provided by the blockchain itself, even if it did carry limitations. (Again, this exists in PoW already in the form of `blockhash`)
+
+## Specification
+Let `entropyhash` for a particuar `blockheight` be a value unknown to any validator before their block and vote are cast. This ensures a validator has to commit to their _possibly-malicious-transactions_ before knowing what the results will be.
+
+I'm not a PoS expert, so I don't know how exactly to generate `entropyhash`, but it seems that hashing `data-used-to-pick-winning-validator` is what we're after. No validator shoud know this ahead of time, and the result of this value should have an economic stake behind it.
```

Comment:
This is much too vague to make an actual standard. If you're looking for ideas, I'd suggest posting your question to the ethereum-magicians forum instead.